release/ray_release/cluster_manager/minimal.py

```diff
@@ -289,3 +289,6 @@ def terminate_cluster(self):
 
     def get_cluster_address(self) -> str:
         return f"anyscale://{self.cluster_name}"
+
+    def get_session_name(self) -> str:
+        return f"anyscale://{self.cluster_name}"
```

Comment:
What is the difference between `get_cluster_address` and `get_session_name`? Can we just use the former?

Reply:
Get cluster address didn’t seem to work with job submission. Not sure if it is a bug or intended. I will ask to the prod team